src/Bitmap/BitmapFile.cpp

```diff
@@ -61,7 +62,7 @@ void BitmapFile::VerifyPixelSizeMatchesImageDimensionsWithPitch() const
 void BitmapFile::VerifyPixelSizeMatchesImageDimensionsWithPitch(uint16_t bitCount, int32_t width, int32_t height, std::size_t pixelsWithPitchSize)
 {
 	if (pixelsWithPitchSize != ImageHeader::CalculatePitch(bitCount, width) * std::abs(height)) {
-		throw std::runtime_error("The size of pixels does not match the image's height time pitch");
+		throw std::runtime_error("The size of pixels does not match the image's height times pitch");
```

Comment:
Very technically accurate. If an end user sees this, or many developers, they may have a weak understanding of what `pitch` is. When I first read this, I mis-read pitch to mean bit depth. I thought it was a small mistake where width had been forgotten. I'm not sure I have any suggestions for how to improve, just wanted to mention this might be easily misunderstood.